internal/pkg/template/workload.go

```diff
@@ -532,6 +533,9 @@ func envControllerParameters(o WorkloadOpts) []string {
 	if o.Storage != nil && o.Storage.requiresEFSCreation() {
 		parameters = append(parameters, "EFSWorkloads,")
 	}
+	if o.NLB != nil {
+		parameters = append(parameters, "NLB") // Invoke env controller so that the env controller action is invoked, retrieving the exported HTTPSCert ARN.
+	}
```

Comment:
Just curious is this just to force an update to the env controller so that it can have the latest output?

Reply:
yes

Reply:
Gotcha. For extensibility concern (like we don't need to add more param like this for the other features merely to trigger an update), I wonder if we should do similar thing as https://github.com/aws/copilot-cli/blob/1fbeceb3647e5711d787368ef2f0ec6ceca96a68/internal/pkg/template/templates/workloads/partials/cf/autoscaling.yml#L13